src/i18n/locales/en.json

```
@@ -140,11 +140,13 @@
     "index_issues_description": "The lucene indexes can degrade or drift from Dependency Track database over time. Even though DT does its best to minimize the drift, the administrative features below are provided to check or restore the indexes if need be. It must be used with caution.",
     "index_rebuild_description": "You can selectively trigger an immediate rebuild of some or all indexes. The index rebuild will be perform by an asynchronous tasks. You can check the progress using Dependency Track logs.",
     "index_use_cases": "The full-text search feature is principally used for the search API (i.e. all the indexes) and internal analyzer fuzzy matching on CPE (i.e. the vulnerable software index).",
+    "integration_auto_create_enabled": "Allow users to create missing projects",
     "integration_defectdojo_enable": "Enable DefectDojo integration",
     "integration_defectdojo_reimport_enable": "Enable reimport",
     "integration_fortify_ssc_enable": "Enable Fortify SSC integration",
     "integration_kenna_connector_id": "Connector ID",
     "integration_kenna_enable": "Enable Kenna Security integration",
+    "integration_sbom_enabled": "Enable SBOM generation",
```

Seeing these changes makes me think that:
- my `gitlab_integration_enabled` change should have been `integration_gitlab_enabled`.
- `integration_sbom_enabled` should be scoped to gitlab, and maybe with the other gitlab options since the sbom upload feature is locked behind the gitlab integration? Maybe even `gitlab_sbom_push_enabled` if we want to match the api config entry. 

First item isn't on you to fix unless you want to, I can make a branch for that change.

```suggestion
    "gitlab_sbom_push_enabled": "Enable Gitlab SBOM push", 
```